pyne/mesh.py

```diff
@@ -1008,9 +1046,20 @@ def tag(self, name, value=None, tagtype=None, doc=None, size=None,
                                  'or dtype'.format(name))
             else:
                 tagtype = MetadataTag
+
         if tagtype is NativeMeshTag or tagtype.lower() == 'nat_mesh':
+            # storage_type works for only NativeMeshTag
```

Comment:
Along the lines of this comment. I'm wondering if we would raise a warning for a non-default value of `storage_type` if the specified tag type isn't `NativeMeshTag`?

Reply:
Done in https://github.com/pyne/pyne/pull/1341/commits/e9020ad9f10e21d928064808cad076131b064b1d.